redis/redis.go

```diff
@@ -211,3 +211,102 @@ type LatencyHistory struct {
 	// ExecutationTime is the amount of time needed for the command execution.
 	ExecutionTime time.Duration
 }
+
+// ClientFlags is redis-server client flags, copy from redis/src/server.h (redis 8.2)
+type ClientFlags uint64
+
+const (
+	ClientSlave                 ClientFlags = 1 << 0  /* This client is a replica */
+	ClientMaster                ClientFlags = 1 << 1  /* This client is a master */
+	ClientMonitor               ClientFlags = 1 << 2  /* This client is a slave monitor, see MONITOR */
```

Comment:
suggestion: use idiomatic iota

```suggestion
	ClientSlave                 ClientFlags = 1 << iota  /* This client is a replica */
	ClientMaster                ClientFlags =         /* This client is a master */
	ClientMonitor               ClientFlags =         /* This client is a slave monitor, see MONITOR */
```
More below